.circleci/config.yml

```diff
@@ -123,6 +187,16 @@ jobs:
       - image: wordpressdevelop/php:7.3-fpm
       - image: *db_image
 
+  php73-core-tests-mu-plugins:
+    <<: *php_core_job
+    environment:
+      - INCLUDE_MU_TESTS: "1"
+      - WP_MULTISITE: "0"
```

Comment:
We should run a multisite version of this too.

Reply:
Done